tests/jax/components/building/distributor_test.py

```diff
@@ -119,27 +119,34 @@ def test_on_building_program_nodes_multi_process(
         "evaluator",
         "trainer",
     ]
+
```

Comment:
see above (line 50 and elsewhere). Think the docstring should change from "to test no multi_process" to "to test single-process"